src/System.Drawing.Common/src/System/Drawing/Font.Unix.cs

```diff
@@ -224,27 +84,20 @@ public static Font FromHfont(IntPtr hfont)
                 newStyle |= FontStyle.Bold;
             }
 
-            if (lf.lfHeight < 0)
-            {
-                newSize = lf.lfHeight * -1;
-            }
-            else
-            {
-                newSize = lf.lfHeight;
-            }
-
-            return (new Font(newObject, lf.lfFaceName, newStyle, newSize));
+            return new Font(newObject, lf.lfFaceName, newStyle, Math.Abs(lf.lfHeight));
         }
 
         public IntPtr ToHfont()
         {
-            if (fontObject == IntPtr.Zero)
-                throw new ArgumentException("Object has been disposed.");
+            if (_nativeFont == IntPtr.Zero)
+            {
+                throw new ArgumentException(SR.ObjectDisposed);
+            }
 
-            return fontObject;
+            return _nativeFont;
         }
 
-        internal Font(IntPtr newFontObject, string familyName, FontStyle style, float size)
+        internal Font(IntPtr new_nativeFont, string familyName, FontStyle style, float size)
```

Comment:
Nit: this is a strange name for an argument... we usually don't use underscores like this.

Reply:
Fixed in https://github.com/dotnet/corefx/pull/29673